vuepress/docs/next/docs/consume/entando-apis.md

```diff
@@ -3,139 +3,80 @@ sidebarDepth: 2
 
-3. Click on `Clients` on the left bar and select your client (e.g. `quickstart-server`)
-
-4. Update the following values under `Settings`:
-    - Set `Valid Redirect URIs` to `http://localhost:[your port]/entando-de-app/*` or `*` to allow all redirect URIs.
-    - Set `Web Origins` to `http://localhost:[your port]` or `*` to accept all origins.
-
-### Start the Entando Application
-
-1.  Clone the Entando reference application:
-
-        git clone https://github.com/entando/entando-de-app
-
-2.  Start the application with the following options: 
-    
-    - Enable the Swagger profile via `-Dspring.profiles.active=swagger`
-    - Enable the Keycloak profile via `-Pkeycloak`
-    - Configure the application connection to Keycloak itself. For simplicity this uses the same client credentials you'll use to try out the APIs.
-       - Set `-Dkeycloak.auth.url` to your Keycloak endpoint (including `/auth`), e.g. `-Dkeycloak.auth.url=http://my-keycloak-server/auth`   
-       - Set `-Dkeycloak.client.id` to your client id, e.g. `-Dkeycloak.client.id=quickstart-server`
-       - Set `-Dkeycloak.client.secret` to your client secret, e.g. `-Dkeycloak.client.secret=my-secret`. See [How to find your client secret](#how-to-find-your-client-secret) above.
-    - (Optional) Set`-Djetty.port=8085` if the default port 8080 is already in use. 
-    - (Optional) To skip the docker steps (or if you don't have docker installed/running), add `-DskipDocker=true`
-    
-    Here's a full example:
-    
-        mvn clean package jetty:run-war -Pjetty-local -Pderby -Pkeycloak -Dspring.profiles.active=swagger -Djetty.port=8085 -Dorg.slf4j.simpleLogger.log.org.eclipse.jetty.annotations.AnnotationParser=error -Dkeycloak.auth.url=http://my-keycloak-host/auth -Dkeycloak.client.id=quickstart-server -Dkeycloak.client.secret=my-client-secret -DskipDocker=true
-
-3.  Wait for the application to start.
-
-        [INFO] Started ServerConnector@1355c8be{HTTP/1.1, (http/1.1)}{0.0.0.0:8085}
-        [INFO] Started @66257ms
-        [INFO] Started Jetty Server
-
-4. Navigate to the Swagger UI in a browser at `/entando-de-app/api/swagger-ui.html`
-
-        http://localhost:[your port]/entando-de-app/api/swagger-ui.html
-    
-
-## APIs Overview
+4. To enable the Swagger UI, add the "SPRING_PROFILES_ACTIVE" environment variable. If the variable is already present, add `swagger` to its comma-delimited value list:
+```
+        - name: SPRING_PROFILES_ACTIVE
+          value: default,swagger
+```
 
-The Entando core exposes REST APIs for every action that can be taken in
-the App Builder environment. For example, you can use
-these APIs to create pages, create page templates or to add widgets to
-pages. The APIs can be used to support automation, testing, or
-integrations with external systems.
+5. (Optional) Reset the deployment `spec.replicas` back to 1 if it was scaled to "0" in the previous steps. Save the deployment to update.
```

Comment:
"..scaled to 0 **for** the previous **steps**" or "scaled to 0 **in** a previous **step**" (it is not scaled to 0 in each of the previous steps, which is how this reads)

0 or "0"?? in step 2 there are no quotes...